src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -217,16 +217,27 @@ export class ChartReleasesComponent implements AfterViewInit, OnInit, OnDestroy
             this.chartItems.clear();
             this.showLoadStatus(EmptyType.Errors);
           } else {
-            this.appService.getChartReleases().pipe(untilDestroyed(this)).subscribe((charts) => {
-              this.chartItems.clear();
+            try {
+              this.appService.getChartReleases().pipe(
+                catchError(() => of(this.showLoadStatus(EmptyType.Errors))),
+                untilDestroyed(this),
+              ).subscribe((charts) => {
+                if (!charts) {
```

Comment:
This could also just mean there are no charts to show instead of having an error.

Reply:
if charts will be an empty array, that won't cause issues 
If it's null or undefined -> then it's error probably?